keepassxc-browser/content/define.js

```diff
@@ -191,7 +191,7 @@ kpxcDefine.markAllPasswordFields = function(chooser) {
         const field = e.currentTarget;
         kpxcDefine.selection.password = field.getAttribute('kpxc-id');
         field.classList.add('kpxcDefine-fixed-password-field');
-        field.innerHTML = 'Password';
+        field.textContent = 'Password';
```

Comment:
Yeah, always get rid of `innerHTML`. AMO's security check will like you… :smile: :+1:

Reply:
Indeed. I learned this the hard way..